draft-ietf-quic-transport.md

```diff
@@ -1160,33 +1160,31 @@ TRANSPORT_PARAMETER_ERROR.
 
 An endpoint MAY use the following transport parameters:
 
-initial_max_stream_id_bidi (0x0002):
-
-: The initial maximum stream ID parameter contains the initial maximum stream
-  number the peer may initiate for bidirectional streams, encoded as an unsigned
-  32-bit integer.  This value MUST be a valid bidirectional stream ID for a
-  peer-initiated stream (that is, the two least significant bits are set to 0 by
-  a server and to 1 by a client).  If an invalid value is provided, the
-  recipient MUST generate a connection error of type TRANSPORT_PARAMETER_ERROR.
-  Setting this parameter is equivalent to sending a MAX_STREAM_ID
-  ({{frame-max-stream-id}}) immediately after completing the handshake.  The
-  maximum bidirectional stream ID is set to 0 if this parameter is absent,
-  preventing the creation of new bidirectional streams until a MAX_STREAM_ID
-  frame is sent.  Note that a default value of 0 does not prevent the
-  cryptographic handshake stream (that is, stream 0) from being used.
+initial_max_streams_bidi (0x0002):
+
+: The initial maximum bidirectional streams parameter contains the initial
+  maximum number of application-owned bidirectional streams the peer may
+  initiate, encoded as an unsigned 16-bit integer.  If this parameter is absent
```

Comment:
I'd prefer 32 bit here.

Reply:
Can you explain why?  Is there a situation in which you'd like to allow that many streams without receiving a MAX_STREAM_ID frame?  @martinthomson suggested reducing the size, and 16 bits still seems quite large for an initial value.

Reply:
You're right. I thought that there's no point in saving bytes here, because I overlooked that the transport parameters are sent by the server as well. So it actually makes sense to shave of 4 bytes (in total) here.